modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc

```diff
@@ -405,11 +396,11 @@ IMAP (OC_User_IMAP), SMB (OC_User_SMB), and FTP (OC_User_FTP).
   ],
 ....
 
-=== Define a custom link to reset password.
-
+=== Define a custom link to reset password
```

Comment:
This issn't quite correct. I'd change it to: "Define a custom link for resetting passwords".